packages/verification-vue-component/src/components/Verification/VerificationItem.vue

```diff
@@ -3,7 +3,7 @@
          :class="{ 'confirmation-step': digitalTwin.confirmationStep, 'error': digitalTwinInformation.error }">
         <div class="card-container">
             <div v-if="digitalTwinInformation.active" class="header">
-                <img src="../../assets/img/certifaction_logo.svg" alt="Certifaction"/>
+                <img :src="headerLogo" alt="logo"/>
```

Comment:
Have you tried switching to CSS for the logo? With CSS it would be at one place to change the theme, not multiple.

Reply:
What did you try? I would expect this would work:
```
:root {
    --cf-verification-logo-url: url('path/to/logo.svg');
}

background-image: var(--cf-verification-logo-url);
```
Maybe it would also make sense to specify the `width` and `height` with CSS variables.